src/pharmpy/tools/structsearch/pkpd.py

```diff
@@ -95,48 +95,51 @@ def create_pkpd_models(
         elif modeltype == 'indirect':
             for argument in list(product(pd_types, [True, False])):
                 pkpd_model = add_indirect_effect(model, *argument)
-                pkpd_model = pkpd_model.replace(
-                    description=f"{modeltype}_{argument[0]}_prod={argument[1]}"
-                )
+                if argument[1]:
```

Comment:
Small thing but to me it's a little bit unclear what `argument` is, it's a tuple right? Consisting of a PD type and a boolean for whether it should be added as production or degradation? Perhaps it would be clearer to, in the for-loop, have `for pd_type, as_production in list(product(pd_types, [True, False]))` so your condition here could be `if as_production`